packages/models/proto/v2/tenant/events.proto

```diff
@@ -52,7 +52,7 @@ message TenantVerifiedAttributeExtensionUpdatedV2 {
   TenantV2 tenant = 2;
 }
 
-message MaintenanceTenantDeletedV2 {
+message MaintenanceTenantDeleteV2 {
```

Comment:
the naming convention is to name events in past participle (same for other occurrences)
```suggestion
message MaintenanceTenantDeletedV2 {
```